sections/trading.tex

```diff
@@ -8,14 +8,15 @@
 The \pagelink{Trading Post}{Trading Post Field} and other effects allow you to either:
 \begin{itemize}
   \item \pagetarget{Trading}{trade}\index{Trading} multiple Resources with the game in accordance to the \hyperlink{Trade Table}{table at the back cover},
-  \item \textbf{remove a card} from your hand at the trading post to gain 1 \svg{gold},
+  \item \textbf{remove exactly one card} from your hand at the trading post to gain 1 \svg{gold},
```

Comment:
```suggestion
  \item \textbf{remove one card} from your hand at the trading post to gain 1 \svg{gold},
```
To remove a card means exactly that - exactly one card. No reason to be overly explicit.